cmd/fetchbranches/branch.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"regexp"
+	"strings"
+)
+
+// Branch is the structure providing specific parameter for the GetBranch execution
+type Branch struct {
+	Repo string
+	URL  string
+}
+
+// Valid tag names
+var tags = []string{"proxy", "shim", "tests", "hyperstart", "runtime"}
+
+// GetBranch retrieves the branches from the pull request
```

Comment:
This comment is rather confusing as the function parameter is `comment`, not `pullRequest`.